zone/lua_mod.cpp

```diff
@@ -862,4 +863,68 @@ void LuaMod::HealDamage(Mob *self, Mob* caster, uint64 value, uint16 spell_id, u
 		lua_pop(L, n);
 	}
 }
+
+void LuaMod::ResistSpellRoll(Mob *self, Mob* caster, int roll, int roll_max, int resist_chance, uint8 resist_type, uint16 spell_id, bool use_resist_override, int resist_override, bool is_charisma_check, bool is_charm_tick, bool is_root, int level_override, int resist_modifier, int &return_value, bool &ignore_default)
+{
+	int start = lua_gettop(L);
+
+	try {
+		if (!m_has_resist_spell_roll) {
+			return;
+		}
```

Comment:
Why doesn't this check higher, as in the very top of the function. I noticed this with other mod calls. If we don't have this mod loaded there's no reason to make a call to `lua_gettop`